configs/components/fesom/fesom-2.1.yaml

```diff
@@ -143,7 +144,6 @@ restart_in_files:
         ice_restart: ice_restart
         par_oce_restart: par_oce_restart
         par_ice_restart: par_ice_restart
-
```

Comment:
can I have that new line back please? :) less differences is better for us, the poor maintainers :)